core/templates/pages/exploration-player-page/layout-directives/audio-bar.directive.html

```diff
@@ -1,12 +1,25 @@
 <div class="audio-header" ng-if="isAudioBarAvailable()">
   <div ng-if="audioBarIsExpanded" class="audio-controls">
-    <div style="width: 560px;">
-      <i class="fas audio-controls-button-icon"
-         ng-click="onPlayButtonClicked()"
-         ng-class="{'fa-ellipsis-h': audioLoadingIndicatorIsShown, 'fa-play-circle': !isAudioPlaying(), 'fa-pause-circle': isAudioPlaying(), 'audio-controls-audio-not-available': !isAudioAvailableInCurrentLanguage() || audioIsLoading}"
-         uib-tooltip="<[!isAudioAvailableInCurrentLanguage() ? ('I18N_PLAYER_AUDIO_NOT_AVAILABLE_IN' | translate:{languageDescription:getCurrentAudioLanguageDescription()}) : '']>"
-         tooltip-append-to-body="true" tooltip-placement="right">
-      </i>
+    <div class="oppia-audio-header-control-buttons">
+      <div class="audio-header-control-icons">
+        <a class="oppia-replay-white-circle" ng-click="onBackwardButtonClicked()" ng-if="progressBarIsShown">
```

Comment:
Any specific reason for having different HTML structure for the forward and backward button?